### ISSUE TITLE
https://godot-kotl.in/ is blocked in Russia

### ISSUE THREAD

**Issue by ih54r94** [raniejade/godot-kotlin#66]
https://godot-kotl.in/ is blocked in Russia by censorship. Is there a mirror?

**Comment by kortov** [comment]
site works for me in Russia and I see no problems 
![image](https://user-images.githubusercontent.com/6348215/76843618-802a8300-6855-11ea-8bd4-285e76a8ca1e.png)

http://r.h-t.co/ru/3/095087c2-3368-ea11-aa75-0003ff73517a

**Comment by ih54r94** [comment]
![изображение](https://user-images.githubusercontent.com/37707290/76844463-5e79cd80-684e-11ea-8448-19955439b04b.png)

**Comment by raniejade** [comment]
Not sure how I can go around with that, the website is hosted by netlify.

**Comment by kortov** [comment]
Yep I got some problems with netlify blocking in Russia. Use opera vpn and other vpns, what can I suggest)
